src/libraries/UtilsLib.sol

```diff
@@ -0,0 +1,28 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+pragma solidity ^0.8.0;
+
+import {ErrorsLib} from "../libraries/ErrorsLib.sol";
+
+/// @title UtilsLib
+/// @author Morpho Labs
+/// @custom:contact security@morpho.org
+/// @notice Library exposing helpers.
+/// @dev Inspired by https://github.com/morpho-org/morpho-utils.
+library UtilsLib {
```

Comment:
note that we could import `toUint128` and `min` from Blue's Utils (but i'm not sure it's better)

Reply:
Had this at first but when I added `saturatingAdd` it seemed cleaner to have a small dedicated lib.

Reply:
just realized that elsewhere we would have called it "Uint128MaxCeilAdd" ("ZeroFloorSub")